google/cloud/documentai_toolbox/wrappers/document.py

```diff
@@ -765,3 +765,33 @@ def export_images(
             index += 1
 
         return output_filenames
+
+    def export_hocr_str(self, filename: str) -> str:
+        r"""Exports a string hOCR version of the Document.
+
+        Args:
+            filename (str):
+                Required. The filename of the original file.
```

Comment:
You need the filename for the hOCR, i don't believe the DocAI response has any filename or something close to it so that's why we ask the caller for that information.

Reply:
In the current code, `filename` is used for `title`, is it the "title" of the whole hOCR document?  If so perhaps we should just call it `title`?  Also if it belongs only to the top level (the whole hOCR document), perhaps it does not need to be passed to the lower level elements (e.g. hOCR page).

Reply:
Will do, i don't think filename gets passed down to lower elements.

Reply:
Just checked, ocr_page needs a title also so that's why it's passed down to page